Rakefile.new

```diff
@@ -110,6 +130,16 @@ task :student do
   Rake::Task["build"].execute
 end
 
+desc "Full Student Skytap VM Build"
+task :student_skytap do
+  cputs "Building Student VM"
+  Rake::Task["init"].execute
+  Rake::Task["student_pre"].execute
+  Rake::Task["build"].execute
+  Rake::Task["skytap_post"].execute
```

Comment:
student_pre and skytap_post both do the supersede search thing. Might be ok to remove that from the skytap_post task perhaps?